app/models/knight.rb

```diff
@@ -1,3 +1,17 @@
 class Knight < Piece
 
+	def is_valid?(new_x, new_y)

```

Comment:
We need to check the case when the new_x and new_y are outside of the board as well.. This is already implemented in the Piece.rb and can be called via `super` like on valid_move for the King.